anomalib/models/cflow/lightning_model.py

```diff
@@ -30,9 +34,7 @@ class Cflow(AnomalyModule):
     def __init__(
         self,
         input_size: Tuple[int, int],
-        backbone: str,
-        layers: List[str],
-        pre_trained: bool = True,
+        feature_extractor: Union[TimmFeatureExtractorParams, TorchFXFeatureExtractorParams],
```

Comment:
a minor suggestion (but it helps to keep things consistent as the code scales): there could be something like

```python
FeatureExtractorParamsTypes = Union[TimmFeatureExtractorParams, TorchFXFeatureExtractorParams]
```

to avoid this double-import-and-union

Reply:
That's a good idea

Reply:
I think it's nice good practice to follow and cheap to implement.
Maybe an exception in mypy checks?